docs/mkdocs.yml

```diff
@@ -46,7 +46,8 @@ nav:
           - zamba.models.model_manager: "api-reference/models-model_manager.md"
           - zamba.models.slowfast_models: "api-reference/models-slowfast_models.md"
           - zamba.models.utils: "api-reference/models-utils.md"
-          - zamba.models.yolox_models: "api-reference/models-yolox_models.md"
+      - zamba.object_detection:
```

Comment:
shouldn't `megadetector_lite_yolox.md` be in object detection as well?

Reply:
I wasn't exactly sure how to arrange that. The `megadetector_lite_yolox` module does get brought into the documentation here https://deploy-preview-167--silly-keller-664934.netlify.app/api-reference/object_detection-yolox/#zamba.object_detection.yolox.megadetector_lite_yolox


But maybe adding a section would give it its own page?

Reply:
my point was that this line creates a broken link: https://github.com/drivendataorg/zamba/blob/a181700b8b814a898901f55e3f0775e7dd1b63e7/docs/mkdocs.yml#L45